src/main/java/sirius/biz/mongo/SortField.java

```diff
@@ -60,6 +60,10 @@ public SortField(MongoEntity owner) {
      * @return the normalized text in lower-case
      */
     public static String normalizeText(String text) {
+        if (Strings.isEmpty(text)) {
+            return null;
```

Comment:
Can this null value lead to problems further down the road?

Reply:
The method crashed before, so it wasn't a valid input anyway, now it is and as a defined result.